packages/core/framework/src/http/utils/get-query-config.ts

```diff
@@ -174,8 +136,8 @@ export function prepareListQuery<T extends RequestQueryFields, TEntity>(
     }
 
     if (
-      queryConfig?.allowedFields?.length &&
-      !queryConfig?.allowedFields.includes(orderField)
+      queryConfig?.allowed?.length &&
+      !queryConfig?.allowed.includes(orderField)
```

Comment:
```suggestion
     allowed.length &&
      !allowed.includes(orderField)
```

Reply:
can drop the `allowed.length` as its defaulting to an array

Reply:
Done

Reply:
no we shouldnt drop the length as otherwise it will never include the order fields when no allowed are provided, instead when nothing is provided we don't want to check it

Reply:
Wow! I wasted 2hours on this. Why did I not read your comment 🤦‍♂️